src/python/pants/backend/python/rules/pytest_runner.py

```diff
@@ -188,35 +191,35 @@ async def setup_pytest_for_target(
     )
 
 
+# TODO(#10618): Once this is fixed, move `TestTargetSetup` into an `await Get` so that we only set
+#  up the test if it isn't skipped.
 @rule(desc="Run Pytest")
 async def run_python_test(
     field_set: PythonTestFieldSet,
-    test_setup: TestTargetSetup,
+    setup: TestTargetSetup,
     global_options: GlobalOptions,
     test_subsystem: TestSubsystem,
 ) -> TestResult:
-    """Runs pytest for one target."""
-    output_files = []
+    if field_set.is_conftest():
+        return TestResult(Status.SKIPPED, "", "", None, None, field_set.address.spec)
```

Comment:
Will this cause something odd to be displayed to the user?

Reply:
No, because `test.py` is updated to handle that. I'll refactor this to be more clear.